### ISSUE TITLE
Cannot set GRPC Max message size annotation: ValueError

### ISSUE THREAD

**Issue by thecooltechguy** [SeldonIO/seldon-core#1165]
Hi,

When I try to set the grpc max message annotation of the yaml of a SeldonDeployment, I'm getting this issue:

```
Traceback (most recent call last):
  File "/usr/local/bin/seldon-core-microservice", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.7/site-packages/seldon_core/microservice.py", line 370, in main
    start_servers(server1_func, server2_func)
  File "/usr/local/lib/python3.7/site-packages/seldon_core/microservice.py", line 44, in start_servers
    target1()
  File "/usr/local/lib/python3.7/site-packages/seldon_core/microservice.py", line 341, in grpc_prediction_server
    user_object, annotations=annotations, trace_interceptor=interceptor
  File "/usr/local/lib/python3.7/site-packages/seldon_core/wrapper.py", line 171, in get_grpc_server
    max_msg = int(annotations[ANNOTATION_GRPC_MAX_MSG_SIZE])
ValueError: invalid literal for int() with base 10: '"20000000"'
```
My annotation looks like
```yaml
"seldon.io/grpc-max-message-size": "20000000"
```

Does anyone know how I can resolve this? It appears that support for the grpc max message size was already added to seldon-core.

**Comment by cliveseldon** [comment]
That's strange. It looks like its passing the quotes as part of the string.
Have you tried: https://docs.seldon.io/projects/seldon-core/en/latest/graph/model_rest_grpc_settings.html

**Comment by cliveseldon** [comment]
Closing this. please reopen if still an issue.

**Comment by gmattar** [comment]
I got this as well on version 1.0.2

```
2020-04-09 23:50:12,982 - seldon_core.microservice:load_annotations:114 - INFO:  Found annotation seldon.io/grpc-max-message-size:"10000000"
2020-04-09 23:50:12,982 - seldon_core.microservice:load_annotations:114 - INFO:  Found annotation seldon.io/grpc-read-timeout:"60000"
2020-04-09 23:50:12,982 - seldon_core.microservice:load_annotations:114 - INFO:  Found annotation seldon.io/grpc-timeout:"60000"
2020-04-09 23:50:12,982 - seldon_core.microservice:load_annotations:114 - INFO:  Found annotation seldon.io/rest-connection-timeout:"60000"
2020-04-09 23:50:12,982 - seldon_core.microservice:load_annotations:114 - INFO:  Found annotation seldon.io/rest-read-timeout:"60000"
2020-04-09 23:50:12,982 - seldon_core.microservice:load_annotations:114 - INFO:  Found annotation seldon.io/rest-timeout:"60000"
2020-04-09 23:50:12,982 - seldon_core.microservice:main:257 - INFO:  Annotations: {'kubernetes.io/config.seen': '"2020-04-09T23:46:55.822939Z"', 'kubernetes.io/config.source': '"api"', 'prometheus.io/path': '"prometheus"', 'prometheus.io/port': '"8000"', 'prometheus.io/scrape': '"true"', 'seldon.io/engine-java-opts': '"-server -Dspring.task.execution.pool.core-size=25 -Dspring.task.execution.pool.max-size=100 -Dspring.task.execution.pool.queue-capacity=1"', 'seldon.io/grpc-max-message-size': '"10000000"', 'seldon.io/grpc-read-timeout': '"60000"', 'seldon.io/grpc-timeout': '"60000"', 'seldon.io/rest-connection-timeout': '"60000"', 'seldon.io/rest-read-timeout': '"60000"', 'seldon.io/rest-timeout': '"60000"'}
2020-04-09 23:50:13,529 - seldon_core.microservice:main:369 - INFO:  Starting servers
Traceback (most recent call last):
  File "/opt/conda/bin/seldon-core-microservice", line 11, in <module>
    sys.exit(main())
  File "/opt/conda/lib/python3.7/site-packages/seldon_core/microservice.py", line 370, in main
    start_servers(server1_func, server2_func)
  File "/opt/conda/lib/python3.7/site-packages/seldon_core/microservice.py", line 44, in start_servers
    target1()
  File "/opt/conda/lib/python3.7/site-packages/seldon_core/microservice.py", line 341, in grpc_prediction_server
    user_object, annotations=annotations, trace_interceptor=interceptor
  File "/opt/conda/lib/python3.7/site-packages/seldon_core/wrapper.py", line 180, in get_grpc_server
    max_msg = int(annotations[ANNOTATION_GRPC_MAX_MSG_SIZE])
ValueError: invalid literal for int() with base 10: '"10000000"'
```

**Comment by cliveseldon** [comment]
Can you try the above example referenced in the docs?
Also which version of Seldon Core are you running.

**Comment by gmattar** [comment]
The example works. 
I'm using a custom image with 1.0.2 version based in https://docs.seldon.io/projects/seldon-core/en/latest/python/python_wrapping_docker.html and calling `seldon-core-microservice` while the image in the example uses a local microservice.py file.
The function `load_annotations` is different between the example and the one available in 1.0.2.
I think it was fixed in https://github.com/SeldonIO/seldon-core/commit/eb0f185d29e84033785b1e473952ebe2f86dc90c#diff-f769fa6147c4959661fb577fdaa76723R115 but I don't get how the fix is available in an old image like seldonio/mock_classifier_grpc:1.0 but was commited 2 months ago and is not even in 1.0.2.

Do you have any suggestion on how to get the fix in my image? is there a way to install the snapshot version with pip?

**Comment by cliveseldon** [comment]
OK yes. So to include that fix you would need to use the latest 1.0.3-SNAPSHOT versions of the python s2i wrappers or install seldon-core python library from master source.

For testing you can try 1.5 version of https://hub.docker.com/repository/docker/seldonio/mock_classifier_grpc
